### PR TITLE
Make the code less ugly!

### PR DESCRIPTION
**Pull request by benjamin-cates**

### CODE REVIEW

File: classification/dataset.py [hunk 1]
```diff
@@ -5,45 +5,14 @@
 import torch.nn.functional as F
 from torchaudio import transforms as audtr
 import numpy as np
-import argparse
 import os
 from functools import partial
 from data_aug.mixup import FastCollateMixup
 
-device = 'cuda' if torch.cuda.is_available() else 'cpu'
```
Comment (benjamin-cates) [inline classification/dataset.py:12]: Line should have stayed here

**Review by Sean1572 — COMMENTED** [review]
mixup.py might need to be spuilt, mixup code should stay in mixup.py and data augmentations should be put into a seperate file under the same folder

File: classification/default_parser.py
Comment (Sean1572) [inline classification/default_parser.py:1]: Excellent change

**Review by Sean1572 — APPROVED** [review]
If code was able to run this changes look good

**Pull request merged**